task_sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -189,6 +192,9 @@ def run(ti: RuntimeTaskInstance, log: Logger):
         # TODO: Handle TI handle failure
         raise
 
+    if msg:
+        SUPERVISOR_COMMS.send_request(msg=msg, log=log)
```

Comment:
This reminds me: we should rework this class/method to have `log` stored as a attribute so we don't need to pass log in to this class.

(Future work; doing this will make it easier to use this from within XCom or Variable class.)

Reply:
https://github.com/apache/airflow/issues/44615